courtroom.cpp

```diff
@@ -980,15 +980,36 @@ void Courtroom::handle_chatmessage_3()
   QString f_emote = m_chatmessage[EMOTE];
 
   ui_vp_showname_image->show();
+
+  // Check for any of 9 possible ways that showname images are
   QVector<QString> exts = {".png", ".jpg", ".bmp"};
+  // 3 places (in order)
+  // 1. Variant folder in character folder in theme folder
+  // 2. Character folder in folder folder
+  // 3. Character folder
+  QStringList directories = {
+      ao_app->get_theme_variant_path("characters/" + f_char.toLower() +
+                                     "/showname"),
+      ao_app->get_theme_path("characters/" + f_char.toLower() + "/showname"),
```

Comment:
I actually don't suggest doing `toLower()`s here, for the `f_char`, in both of these lines.
By lowercasing the character name, you're basically guaranteeing a regex search on Linux, unless the character folder *was* lowercased.

If ya keep it as it is, it is possible that the capitalisation will match the folder (Windows users generally do pay attention to matching char folder and name cases, surprisingly), and thus the path can immediately be returned as correct.

Reply:
```suggestion
      ao_app->get_theme_path("characters/" + f_char + "/showname"),
```